kernel/src/table_features/mod.rs

```diff
@@ -11,10 +14,24 @@ use delta_kernel_derive::internal_api;
 
 pub(crate) use column_mapping::column_mapping_mode;
 pub use column_mapping::{validate_schema_column_mapping, ColumnMappingMode};
+// Write-side column mapping support
+pub(crate) use column_mapping::{
+    assign_column_mapping_metadata, get_column_mapping_mode_from_properties, get_physical_name,
+    COLUMN_MAPPING_MAX_COLUMN_ID_KEY,
+};
 pub(crate) use timestamp_ntz::validate_timestamp_ntz_feature_support;
 mod column_mapping;
 mod timestamp_ntz;
 
+/// Prefix for table feature override properties.
+/// Properties with this prefix (e.g., `delta.feature.deletionVectors`) are used to
+/// explicitly enable table features in the protocol.
```

Comment:
instead of using "enable" perhaps say: `explicitly turn on support for that feature in the protocol`